### PR TITLE
Use Semantic Versioning

### PR DESCRIPTION
**Pull request by marmoure**
This PR depends on #35 
Switch to `1.1.0-SNAPSHOT` instead of `1.1-SNAPSHOT` which matches all the other version numbers we are using.

### CODE REVIEW

**Comment by adamretter** [comment]
@marmoure Please can you also update the `README.md` file with the correct version number as part of this PR

File: README.md [hunk 1]
```diff
@@ -16,7 +16,7 @@ The [compiled artifact](http://search.maven.org/#search%7Cga%7C1%7Cg%3A%22com.ev
 <dependency>
     <groupId>com.evolvedbinary.cql</groupId>
     <artifactId>corpusql-parser</artifactId>
-    <version>1.0</version>
+    <version>1.1-SNAPSHOT</version>
```
Comment (adamretter) [inline README.md:19]: This should be `1.1.0-SNAPSHOT`!

**Review by adamretter — CHANGES_REQUESTED** [review]
Version number needs fixing

**Comment by marmoure** [comment]
@adamretter it was a wong commit from my laptop
i fixed it now

**Pull request merged**